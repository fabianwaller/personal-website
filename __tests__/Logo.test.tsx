import Logo from "@/components/Logo";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("Logo", () => {
  it("renders Logo unchanged", () => {
    const component = render(<Logo />);
    expect(component.asFragment()).toMatchSnapshot();
  });
  it("renders white Logo unchanged", () => {
    const component = render(<Logo white />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
