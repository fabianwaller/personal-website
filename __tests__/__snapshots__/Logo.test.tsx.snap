// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders Logo unchanged 1`] = `
<DocumentFragment>
  <a
    href="/"
  >
    <span
      class="text-md font-semibold tracking-wide text-title-normal underline-offset-4 hover:text-primary-foreground hover:underline"
    >
      Fabian
    </span>
  </a>
</DocumentFragment>
`;

exports[`Logo renders white Logo unchanged 1`] = `
<DocumentFragment>
  <a
    href="/"
  >
    <span
      class="text-md font-semibold tracking-wide text-title-normal underline-offset-4 hover:text-primary-foreground hover:underline"
    >
      Fabian
    </span>
  </a>
</DocumentFragment>
`;
